pennylane/ops/qubit.py

```diff
@@ -635,6 +780,8 @@ class Hermitian(Observable):
     "CRY",
     "CRZ",
     "CRot",
+    "U2",
+    "U3",
```

Comment:
Should we add `U1` as an alternate name for `PhaseShift`?

Reply:
Yeah let's do it! It can simply call
```python
def decomposition(params, wires):
    return [PhaseShift(params, wires)]
```

Reply:
lol wow you got really excited by this :tada: :smile:

Reply:
This PR excites me. Functional templates are really powerful

Reply:
done